cypress/integration/E2E_tests/ProfileTests/profile.spec.js

```diff
@@ -41,4 +41,27 @@ describe("Profile component can ", () => {
   it("send a message", () => {
     cy.get("[data-cy=message]").click();
   });
+  it("interact with the user portfolio",()=>{
+    //The plan here is to press the new item button and add a stock
+    cy.get("[data-cy=positions-new-item-btn").click();
+    cy.get("[data-cy=positions-new-item-submit-btn]").click();
+    //To edit that stock
+    cy.get("[data-cy=edit-stock-button]").last().click();
+    cy.get("[data-cy=positions-new-item-symbol-tf]").type("GOOG");
+    cy.get("[data-cy=positions-new-item-position-size-tf]").type("10");
```

Comment:
I would make it erase the zero before typing the 10. Also I suppose we can keep this test on this page for coverage, but technically on another user's profile, you should only be able to view their portfolio.

Reply:
This is just to validate that you can edit your portfolio. I will remove the 0 first as you suggest.